.jshintrc

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I've been thinking about adding a jshint for a bit.. @spikebrehm did you have any thoughts against jshint?